spec/memfs/file_spec.rb

```diff
@@ -527,9 +527,16 @@ module MemFs
       end
 
       context 'when the period is the last character in path' do
-        it 'returns an empty string' do
-          extname = described_class.extname('test-subject.')
-          expect(extname).to eq ''
+        if MemFs.ruby_version_gte?('2.7') && !MemFs.windows?
```

Comment:
If I understood correctly, the original `File::extname` is being used by MemFs, not a fake implementation, correct? So we don't have to implement this behavior ourselves.

https://github.com/simonc/memfs/blob/26e0cb91643f98c523352106f135dc0171d86987/lib/memfs/file.rb#L29-L32

Reply:
Exactly yes 😊